src/Illuminate/Mail/TransportManager.php

```diff
@@ -91,7 +91,7 @@ protected function createSesDriver()
     protected function addSesCredentials(array $config)
     {
         if ($config['key'] && $config['secret']) {
-            $config['credentials'] = Arr::only($config, ['key', 'secret']);
+            $config['credentials'] = Arr::only($config, ['key', 'secret', 'token']);
```

Comment:
What happens if `token` is not defined in the array at this point? Does `Arr::only()` handle that gracefully?

Reply:
Sorry, I've been away for the last couple of days for Easter.

From looking at the Arr::only function, all it does is an intersection:

```
return array_intersect_key($array, array_flip((array) $keys));
```

So, running that through PHP with all 3 gives:

![image](https://user-images.githubusercontent.com/1312258/38255328-765a5ba0-3753-11e8-9d15-458f17670681.png)

Just having 'key' and 'secret' in the config gives:

![image](https://user-images.githubusercontent.com/1312258/38255381-95148728-3753-11e8-9abc-33e4fe684532.png)

So it handles the lack of the token gracefully based off the source code. I can check with the actual code tonight once I've finished work, however based off the source code, it is handled gracefully.

Reply:
I can confirm it has no effect on existing functionality. Our current local development environment for the given project only contains key and secret, yet our live servers with the above change contain key, secret and token, and both send emails as expected.

Reply:
Is there any progress on this guys?